docs/wiki/[Examples]-Deploy-ZT-Network.md

```diff
@@ -0,0 +1,59 @@
+# Azure landing zone Terraform deployment with Zero Trust network principles (Hub and Spoke)
+
+This guide will review how to deploy the Azure landing zone Terraform accelerator with a jump start on Zero Trust Networking Principles for Azure landing zones.
+
+For more information on Zero Trust security model and principles visit [Secure networks with Zero Trust](https://learn.microsoft.com/security/zero-trust/deploy/networks).
+
+Deploying Zero Trust network principles with the Terraform deployment will involve setting certain module variables to a value.  Some of these are already the default values, and do not need to be changed.  Others will need to be changed from their default values.
+
+These variables reside in the [variables.tf](../../variables.tf) file and can be provided at run time, or replaced/overridden for your deployment.
+
+## Configure Connectivity resources
+
+In the *configure_connectivity_resources* variable's *ddos_protection_plan* object,  you will set the following:
+
+| Variable | Zero Trust Value | Default Value |
+|---|---|---|
+| enabled | **TRUE** | FALSE |
```

Comment:
```suggestion
| `enabled` | `true` | `false` |
```
@brsteph can you repeat this format throughout please :)

Reply:
Adjusting, thank you for pointing this out!